tests/apps/search/test_autocomplete_categories.py

```diff
@@ -89,11 +89,7 @@ def execute_query(self, querystring=""):
             }
             for category in categories
         ]
-        bulk(
-            actions=actions,
-            chunk_size=settings.ES_CHUNK_SIZE,
-            client=settings.ES_CLIENT,
-        )
+        bulk(actions=actions, chunk_size=500, client=ES_CLIENT)
```

Comment:
Why using a fixed chunk size instead of the configured/default one?

Reply:
This is a test so I'm testing the value passed as well so that the test would break if someone makes a change that alters the chunk size unwillingly.